src/index.js

```diff
@@ -0,0 +1,208 @@
+import './babel-maybefill';
+
+import _ from 'lodash';
+import spawn from './spawn-promise';
+import asar from 'asar';
+import path from 'path';
+import temp from 'temp';
+import sfs from 'fs';
+
+import { fs } from './promisify';
+
+const d = require('debug')('electron-windows-installer:main');
+const isWindows = process.platform === 'win32';
+
+temp.track();
+
+export function p(strings, ...values) {
+  let newPath = String.raw(strings, ...values);
+  let parts = _.map(newPath.split(/[\\\/]/), (x) => x || '/');
+
+  try {
+    return path.resolve(...parts);
+  } catch(e) {
+    return path.join(...parts);
+  }
+}
+
+function statSyncNoException(file) {
+  try {
+    return sfs.statSync(file);
+  } catch (e) {
+    return null;
+  }
+}
+
+function locateExecutableInPath(exe) {
+  // NB: Windows won't search PATH looking for executables in spawn like
+  // Posix does
+
+  // Files with any directory path don't get this applied
+  if (exe.match(/[\\\/]/)) {
+    d('Path has slash in directory, bailing');
+    return exe;
+  }
+
+  let target = path.join('.', exe);
+  if (statSyncNoException(target)) {
+    d(`Found executable in currect directory: ${target}`);
+    return target;
+  }
+
+  let haystack = process.env.PATH.split(isWindows ? ';' : ':');
```

Comment:
node has a nice helper for PATH delimiter.

https://nodejs.org/api/path.html#path_path_delimiter